src/imports.rs

```diff
@@ -41,19 +42,13 @@ impl<'a> FmtVisitor<'a> {
         // 2 = } + ;
         let used_width = indent + 2;
 
-        let remaining_budget = if used_width >= budget {
-            return None;
-        } else {
-            budget - used_width
-        };
-
```

Comment:
I mean like integer overflow, but at the bottom end. If used_width > budget then remaining_budget will be less than 0, but it is an unsigned integer, so this will cause a run time crash (or a silent error if assertions are disabled).